login.html

```diff
@@ -12,15 +12,18 @@
     <div class="background-login">
       <div class="container-login">
         <img class="login-logo" src="./assets/LogoWolox.png" alt="Wolox logo">
-        <label for="mail">Email</label>
-        <input type="email" name="mail">
-        <label for="pass">Password</label>
-        <input type="password" name="pass">
-        <a href="list.html">
-          <button class="btn-green">Login</button>
-        </a>
-        <hr>
-        <button class="btn-inverted-green">Sing up</button>
+        <div class="form-group">
+          <label for="mail">Email</label>
+          <input type="email" name="mail">
+          <label for="pass">Password</label>
+          <input type="password" name="pass">
+          <a href="list.html" class="btn">
+            <button class="btn-green">Login</button>
```

Comment:
Don't use `a` and `button` together, it's not really accurate in semantics. This should only be a link (`a`)